common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -498,28 +499,24 @@ def test_user_can_revoke_and_refund(self, mock_get_course_runs, mock_refund_enti
         assert CourseEnrollment.is_enrolled(self.user, self.course.id)
 
         # Unenroll with Revoke for refund
-        with patch('lms.djangoapps.commerce.signals.handle_refund_entitlement') as mock_refund_handler:
-            REFUND_ENTITLEMENT.connect(mock_refund_handler)
-
-            # pre_db_changes_entitlement = course_entitlement
-            revoke_url = url + '?is_refund=true'
-            response = self.client.delete(
-                revoke_url,
-                content_type='application/json',
-            )
-            assert response.status_code == 204
-
-            course_entitlement.refresh_from_db()
-            assert mock_refund_handler.called
-            assert (CourseEntitlementSerializer(mock_refund_handler.call_args[1]['course_entitlement']).data ==
-                    CourseEntitlementSerializer(course_entitlement).data)
-            assert not CourseEnrollment.is_enrolled(self.user, self.course.id)
-            assert course_entitlement.enrollment_course_run is None
-            assert course_entitlement.expired_at is not None
+        revoke_url = url + '?is_refund=true'
+        response = self.client.delete(
+            revoke_url,
+            content_type='application/json',
+        )
+        assert response.status_code == 204
+
+        course_entitlement.refresh_from_db()
+        assert mock_refund_entitlement.is_called
+        assert (CourseEntitlementSerializer(mock_refund_entitlement.call_args[1]['course_entitlement']).data ==
+                CourseEntitlementSerializer(course_entitlement).data)
+        assert not CourseEnrollment.is_enrolled(self.user, self.course.id)
+        assert course_entitlement.enrollment_course_run is None
+        assert course_entitlement.expired_at is not None
 
     @patch('entitlements.api.v1.views.CourseEntitlement.is_entitlement_refundable', return_value=False)
-    @patch('lms.djangoapps.commerce.signals.refund_entitlement', return_value=[1])
-    @patch("entitlements.api.v1.views.get_course_runs_for_course")
+    @patch('entitlements.api.v1.views.refund_entitlement', return_value=True)
```

Comment:
Doesn't this live in the commerce utils?

Reply:
Same as above.